src/ui/components/Comments/EditableRemark.tsx

```diff
@@ -51,6 +53,13 @@ export default function EditableRemark({
     setIsEditing(false);
   };
 
+  const handleSaveChanges = (editorState: SerializedEditorState) => {
+    saveChanges(editorState);
+    dismissAddCommentNag();
```

Comment:
I think this call to `dismissAddCommentNag` should be put inside of `saveChanges`

I don't think there's any reason to add a new wrapper function that just does one more unconditional call, especially when `saveChanges` is never called without it.

Reply:
True

Reply:
This hasn't changed.